app/src/androidTest/java/com/teamtwo/apilol/uiTestDI.kt

```diff
@@ -36,6 +39,13 @@ interface UiTestComponent : ChampionsComponent {
 @Module
 class UiTestServerModule {
 
+    @Provides @Singleton
+    fun databaseProvider(app: Application): ApiLolDatabase = Room.databaseBuilder(
+        app,
+        ApiLolDatabase::class.java,
+        "api_lol_db"
+    ).build()
```

Comment:
```suggestion
    fun databaseProvider(app: Application): ApiLolDatabase = Room.inMemoryDatabaseBuilder(
        app,
        ApiLolDatabase::class.java
    )
        .allowMainThreadQueries()
        .build()
```

Según pone en la documentación de Room, es mejor usar una base de datos temporal en memoria para evitar tocar nada de la base de datos real de la aplicación. Además es una implementación más ligera y rápida para acceder.